### PR TITLE
Product route

### PR DESCRIPTION
**Pull request by modrayyes**

### CODE REVIEW

**Review by NadiaJalalTamimi — DISMISSED** [review]

File: server/controllers/getProducts.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const getProductQuery = require('../database/queries/products');
```
Comment (umarsalah) [inline server/controllers/getProducts.js:1]: you export the query function in inde.js as an object
```js
module.exports = {
    getProductsQuery
};
```
So you have to import here an object
```js
const {getProductQuery} = require('../database/queries/products');
```

**Review by umarsalah — CHANGES_REQUESTED** [review]

File: server/controllers/getProducts.js (hunk 1 above)
Comment (LinaYahya) [reply server/controllers/getProducts.js]: Good point @umarsalah, totally agree with you.

**Review by NadiaJalalTamimi — DISMISSED** [review]

**Review by NadiaJalalTamimi — APPROVED** [review]

**Review by ahmadfarid-059 — DISMISSED** [review]

**Review by asmaaamin — APPROVED** [review]

**Review by NadiaJalalTamimi — APPROVED** [review]

**Review by asmaaamin — CHANGES_REQUESTED** [review]
use this controller in the router

**Review by LinaYahya — COMMENTED** [review]
fix eslint errors,
and where is the route?
@modrayyes

**Review by ahmadfarid-059 — DISMISSED** [review]

**Review by NadiaJalalTamimi — APPROVED** [review]

**Comment by modrayyes** [comment]
> use this controller in the router

we did use it .

**Review by asmaaamin — DISMISSED** [review]

**Comment by modrayyes** [comment]
> fix eslint errors, and where is the route? @modrayyes

all fixed , converted to async await , and it's already used in the router by ahmed.

**Review by asmaaamin — APPROVED** [review]

**Review by NadiaJalalTamimi — APPROVED** [review]

**Review by umarsalah — APPROVED** [review]

**Pull request merged**